main_server/client_interface/html/index.php

```diff
@@ -1,43 +1,61 @@
 <?php
 define('ROOTPATH', __DIR__ . '/../..');
 defined('BMLT_EXEC') or define('BMLT_EXEC', 1);
-// define ( '_DEBUG_MODE_', 1 ); //Uncomment for easier JavaScript debugging.
-require_once(dirname(__FILE__).'/bmlt-basic/bmlt_basic.class.php') ?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
-
-    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+require_once('../../local_server/server_admin/c_comdef_admin_main_console.class.php');
+$console_object = new c_comdef_admin_main_console($_REQUEST);
+?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
     <head>
         <meta http-equiv="content-type" content="text/html; charset=utf-8" />
-        <title>Browse the Root Server</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
         <?php
-            $url = dirname(dirname(dirname($_SERVER['PHP_SELF'])));
-            
-        if ($url == '/') {
-            $url = '';
+        $url = dirname(dirname(dirname($_SERVER['PHP_SELF']))) !== "/" ? dirname(dirname(dirname($_SERVER['PHP_SELF']))) : '';
+        $shortcut_icon = "$url/local_server/server_admin/style/images/shortcut.png";
+        $stylesheet = "$url/local_server/server_admin/style/styles.css?v=" . time();
+        $my_observable_service_body_ids = [];
+        foreach ($console_object->my_observable_service_bodies as $observable_service_body_id) {
+            array_push($my_observable_service_body_ids, intval($observable_service_body_id->GetID()));
+        }
+
+        function getBCP47TagForISO631Language($code)
+        {
```

Comment:
If I am reading this function correctly, I think it can be rewritten like this without a loop:

```
$default = 'en-US';

$iso630tobcp47 = [
    'de' => 'de-DE',
    'dk' => 'da-DK',
    'es' => 'es-US',
    'fa' => 'fa-IR',
    'fr' => 'fr-CA',
    'it' => 'it-IT',
    'pl' => 'pl-PL',
    'pt' => 'pt-BR',
    'sv' => 'sv-SE'
];

return array_key_exists($code, $iso630tobcp47) ? $iso630tobcp47[$code] : $default;
```